src/core_codemods/sonar_numpy_nan_equality.py

```diff
@@ -0,0 +1,42 @@
+import libcst as cst
+from codemodder.codemods.base_codemod import Reference
+from codemodder.codemods.libcst_transformer import (
+    LibcstTransformerPipeline,
+)
+from codemodder.codemods.sonar import SonarDetector
+from codemodder.codemods.sonar import SonarCodemod
+
+from core_codemods.api import Metadata
+from core_codemods.numpy_nan_equality import (
+    NumpyNanEquality,
+    NumpyNanEqualityTransformer,
+)
+
+
+class SonarNumpyNanEqualityTransformer(NumpyNanEqualityTransformer):
+    def leave_Comparison(
+        self, original_node: cst.Comparison, updated_node: cst.Comparison
+    ) -> cst.BaseExpression:
+        if self.filter_by_result(self.node_position(original_node)):
+            return super().leave_Comparison(original_node, updated_node)
+        return updated_node
+
+
+rules = ["python:S6725"]
+
+SonarNumpyNanEquality = SonarCodemod(
+    metadata=Metadata(
+        name="numpy-nan-equality-S6725",
+        summary="Sonar: " + NumpyNanEquality.summary,
```

Comment:
Not necessarily for this PR but it seems like there should be some kind of shortcut to avoid all of these references to the core codemod class or else we're going to end up with a lot of repeated code.

Reply:
Yeah, I was figuring out how viable it was to derive the metadata from the original codemod as I went. I'll make it easier for the next codemod. Probably with a new constructor for `SonarCodemod`.